csharp/private/actions/assembly.bzl

```diff
@@ -172,4 +172,5 @@ def AssemblyAction(
         deps = deps,
         transitive_refs = refs,
         transitive_runfiles = runfiles,
+        provided_by_netstandard = False
```

Comment:
Hm, maybe this is an awkard name because `target_framework` could be a netstandard framework...
I'm trying to think of a better name but it's awkward.

Reply:
polyfilled_by_netstandard? that's a word we haven't used so probably not good either.

Reply:
`provided_by_netstandard_but_not_netstandard_itself`
`crossed_the_netstandard_boundary`
`provided_by_alternate_framework_line`

Reply:
`secretly_netstandard` (jk)

Reply:
Alternatively, just record the "actual tfm" and then do the logic in the sort `key=` function, thus avoiding having to name the particular situation